docs/usage/self-hosted-configuration.md

```diff
@@ -919,6 +919,10 @@ Some cloud providers offer services to receive metadata about the current instan
 or [GCP VM metadata](https://cloud.google.com/compute/docs/metadata/overview).
 Use this option to control whether Renovate should try to access these services.
 
+## useSqliteCache
```

Comment:
I would prefer to create a new option `packageCacheTape` to join with redis config.

than we can more easily extend to other options.

**values**
- `redis://..` for refis
- `cacache` (default)
- `sqlite`

Reply:
I would prefer to keep as-is for now because:
- It would require config migration if not done in a major
- Some Mend products have redisUrl logic embedded and not sure we can change that easily

Reply:
the migration would be trivial 🤷‍♂️

Reply:
Also needs CLI and ENV migrations. So I prefer a major

Reply:
ok 🤷‍♂️

Reply:
should we move it to experimental env until next major? so we can more easily update it?